dbt_subprojects/daily_spellbook/models/eigenlayer/v2_withdrawal_queued_flattened.sql

```diff
@@ -0,0 +1,51 @@
+{{ 
+    config(
+        schema = 'eigenlayer',
+        alias = 'v2_withdrawal_queued_flattened',
+        post_hook='{{ expose_spells(\'["ethereum"]\',
+                                    "project",
+                                    "eigenlayer",
+                                    \'["bowenli"]\') }}'
+    )
+}}
+
+
+WITH
+    parsed_data AS (
+        SELECT
+            evt_tx_hash,
+            evt_index,
+            evt_block_time,
+            evt_block_number,
+            withdrawalRoot,
+            JSON_PARSE(withdrawal) AS parsed_withdrawal
+        FROM
+            {{ source('eigenlayer_ethereum', 'DelegationManager_evt_WithdrawalQueued') }}
```

Comment:
May be incremental?

Reply:
i tested it, it only take like 2s in dune trino, so it prob not worth it imho